monkey/tests/unit_tests/infection_monkey/exploit/test_http_agent_binary_request_handler.py

```diff
@@ -45,7 +41,7 @@ def binary_request_1(port) -> AgentBinaryDownloadReservation:
     return AgentBinaryDownloadReservation(
         UUID_1,
         OperatingSystem.LINUX,
-        use_agent_binary,
+        DEFAULT_AGENT_TEMPLATE,
```

Comment:
Is there a test where this parameter is set to `None`?

Reply:
Nope, I will add one.